### PR TITLE
Sofia/invite modal

### PR DESCRIPTION
**Pull request by sofiahowardjimenez**
## Documentation

_Describe your feature & code in detail, so that future developers know how to handle it_

- Added some frontend changes to the Invite Screen to match hi-fi designs
- Updated Member card to handle which icon to show based on status
- Created userInvite.ts in firestore and updated the scheme file
- Added the "Add User" modal and created the invite radio button component for the modal

## Related PRs

_Describe any other PRs that might affect/depend on yours_

## Testing

_List all manual tests done/new unit tests you have implemented_

_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Screenshots

_Required for visual changes_
IMG_8532
IMG_8533
IMG_8534

CC: @jacobjk01 @hukellyy

### CODE REVIEW

File: src/components/InviteRadioButton.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import * as React from "react";
+import { useState } from "react";
+import { StyleSheet, View, Text, Pressable, Image } from "react-native";
+
+const InviteRadioButton = () => {
+  const styles = StyleSheet.create({
```
Comment (jacobjk01) [inline src/components/InviteRadioButton.tsx]: styles in separate file plz! for consistency!

File: src/components/RectangularButton.tsx [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+  onPress,
+  buttonStyle,
+  textStyle,
+}: {
+  text: string;
+  onPress: (event: GestureResponderEvent) => void;
+  buttonStyle: any;
+  textStyle: any;
 }) => {
   const styles = StyleSheet.create({
```
Comment (jacobjk01) [inline src/components/RectangularButton.tsx]: I know its v unnecessary but let's just do separate file to be consistent

Comment (jacobjk01) [reply src/components/RectangularButton.tsx]: That way people who use the codebase know exactly what to expect when they're looking for styling

File: src/screens/Invite/InviteScreen.tsx [hunk 3]
```diff
@@ -1,44 +1,162 @@
...
+          <FormInput
+            name="email"
+            rules={{ required: "Field required" }}
+            label="email address"
+            placeholder="email@gmail.com"
+            keyboardType="email-address"
+          />
+        </FormProvider>
+        <Text style={styles.modalText}>family relationship</Text>
+        <InviteRadioButton />
```
Comment (jacobjk01) [inline src/screens/Invite/InviteScreen.tsx:154]: Will probably need to pass a set function into this later, so that you can access the value you chose

Comment (jacobjk01) [reply src/screens/Invite/InviteScreen.tsx:154]: Just thinking ahead a bit

**Comment by jacobjk01** [comment]
Please also make sure profile and transactions image sizes are ok if u can!

File: src/screens/Invite/InviteScreen.tsx (hunk 3 above)
Comment (sofiahowardjimenez) [reply src/screens/Invite/InviteScreen.tsx:154]: Is it okay if I fix this later? I'm still trying to figure that out. Thanks!

**Pull request merged**